omnibus/config/software/more-ruby-cleanup.rb

```diff
@@ -52,7 +52,6 @@
       diagrams
       example
       examples
-      ext
```

Comment:
Can we figure out how to skip the rubyprof one alone. These are pretty big on the installs and other than this case seem to be useless